### PR TITLE
0.35.0 blog post

### PR DESCRIPTION
**Pull request by devlead**
* Updated Cake tool to version 0.35.0
* v0.35.0 Blog Post

@cake-build/cake-team please review.

### CODE REVIEW

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+- [pritchums](https://github.com/pritchums)
+- [RehanSaeed](https://github.com/RehanSaeed)
+
+Full details of everything that was included in this release can be seen below.
+
+<!--excerpt-->
+
+### .NET Core 3 support
+
+[Cake.Tool](https://www.nuget.org/packages/Cake.Tool/0.35.0) is now addition to .NET Core 2.1 also is compiled for .NET Core 3.0 using .NET Core 3 RTM SDK.
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
[Cake.Tool](https://www.nuget.org/packages/Cake.Tool/0.35.0) is now,  in addition to .NET Core 2.1, also compiled for .NET Core 3.0 using .NET Core 3 RTM SDK.
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+```bash
+[*   ] = Minimal
+[**  ] = Normal
+[*** ] = Verbose
+[****] = Diagnostic
+```
+
+### Runtime identification & compiler constants
+
+Runtime identification and detection has been improved, so now Cake will not just compile scripts as .NET Standard but the actual .NET Core runtime it's running under. This for example means Cake now can fully utilize a NuGet package compiled for .NET Core 3.
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
Runtime identification and detection has been improved, so now Cake will not just compile scripts as .NET Standard but the actual .NET Core runtime it's running under. This for example means Cake can now fully utilize a NuGet package compiled for .NET Core 3.
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+As Cake now will have different language features depending on which .NET Framework / Runtime it's running under we've introduced the same preprocessor symbols you generally get in regular C# with one Cake specific addition.
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
As Cake will now have different language features depending on which .NET Framework / Runtime it's running under we've introduced the same preprocessor symbols you generally get in regular C# with one Cake specific addition.
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 1 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 2 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 3 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+- [pritchums](https://github.com/pritchums)
+- [RehanSaeed](https://github.com/RehanSaeed)
+
+Full details of everything that was included in this release can be seen below.
+
+<!--excerpt-->
+
+### .NET Core 3 support
+
+[Cake.Tool](https://www.nuget.org/packages/Cake.Tool/0.35.0) is now,  in addition to .NET Core 2.1, also compiled for .NET Core 3.0 using .NET Core 3 RTM SDK.
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
[Cake.Tool](https://www.nuget.org/packages/Cake.Tool/0.35.0) is now, in addition to .NET Core 2.1, also compiled for .NET Core 3.0 using .NET Core 3 RTM SDK.
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+### Issues
+
+As part of this release we had [10 issues](https://github.com/cake-build/cake/milestone/60?closed=1) closed.
+
+__Feature__
+
+- [__#2603__](https://github.com/cake-build/cake/issues/2603) Add .NET Core 3 to Cake.Tool update to .NET Core 3 SDK.
+
+__Improvement__
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
__Improvements__
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+- [__#2606__](https://github.com/cake-build/cake/issues/2606) Unable to reference Newtonsoft.Json > 11.0.2.
+- [__#2601__](https://github.com/cake-build/cake/issues/2601) Update Microsoft.CodeAnalysis.CSharp.Scripting to 3.2.1.
+- [__#2599__](https://github.com/cake-build/cake/issues/2599) Update to Autofac 4.9.4.
+- [__#2585__](https://github.com/cake-build/cake/issues/2585) Cake.Tool - How in the world do I run a specific task?.
+
+__Documentation__
+
+- [__#2590__](https://github.com/cake-build/cake/pull/2590) Update confusing GitVersionVerbosity docs.
+
+__Bug__
```
Comment (gep13) [inline input/blog/2019-09-28-cake-v0.35.0-released.md]:
```suggestion
__Bugs__
```

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 4 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 5 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

File: input/blog/2019-09-28-cake-v0.35.0-released.md (hunk 6 above)
Comment (devlead) [reply input/blog/2019-09-28-cake-v0.35.0-released.md]: Fixed.

**Pull request merged**